packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -53,8 +53,10 @@ export function makeOperationGenerator(
 	}
 
 	function exclusiveRange(state: ClientOpState): RangeSpec {
-		const start = startPosition(state);
-		const end = state.random.integer(start + 1, state.channel.getLength());
+		let max = state.channel.getLength() - 2 < 0 ? 0 : state.channel.getLength() - 2;
```

Comment:
The condition to require length > 1 makes these checks unnecessary. It's not causing any harm in terms of remaining bugs though.